### PR TITLE
Explore: Improve timeseries limit disclaimer

### PR DESCRIPTION
**Pull request by Elfo404**
**What is this feature?**

Revamps the "Showing only 20 series" disclaimer in Explore:

Screenshot 2023-09-27 at 18 09 19

**Special notes for your reviewer:**

The code is uglier then my hair right now, and the Graph is coupled with the container, but this should unblock https://github.com/grafana/grafana/pull/75499. Will create a follow up issue to do a tiny bit of refactoring as discussed with @gelicia to introduce a `limit` prop the the ExploreGraph (instead of the current solution) so it'll be a bit cleaner

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/features/explore/Logs/LogsVolumePanel.tsx [hunk 1]
```diff
@@ -80,6 +80,7 @@ export function LogsVolumePanel(props: Props) {
         anchorToZero
         yAxisMaximum={allLogsVolumeMaximum}
         eventBus={props.eventBus}
+        showAllTimeSeries
```
Comment (matyax) [inline public/app/features/explore/Logs/LogsVolumePanel.tsx:83]: Should we also make this a state variable that the user can toggle at some point?

**Review by matyax — COMMENTED** [review]
Logs changes look good 👍

File: public/app/features/explore/Logs/LogsVolumePanel.tsx (hunk 1 above)
Comment (Elfo404) [reply public/app/features/explore/Logs/LogsVolumePanel.tsx:83]: whole context in https://raintank-corp.slack.com/archives/C02TXB92G6T/p1695815888660829

the TLDR is that this is just to unblock som other things, the logs volume panel can only have like 8 series i think (max 1 per each log level) so it's not really affected, we'll refactor this to make it better

File: public/app/features/explore/Graph/GraphContainer.tsx [hunk 2]
```diff
@@ -58,6 +70,18 @@ export const GraphContainer = ({
   return (
     <PanelChrome
       title="Graph"
+      titleItems={[
+        MAX_NUMBER_OF_TIME_SERIES < data.length && !showAllTimeSeries && (
+          <div key="disclaimer" className={styles.timeSeriesDisclaimer}>
+            <Tooltip content="Showing only 20 time series">
```
Comment (gelicia) [inline public/app/features/explore/Graph/GraphContainer.tsx:76]:
```suggestion
            <Tooltip content={`Showing only ${MAX_NUMBER_OF_TIME_SERIES} time series`}>
```

Comment (Elfo404) [reply public/app/features/explore/Graph/GraphContainer.tsx:76]: lol, thanks for catching this

**Review by gelicia — APPROVED** [review]

**Pull request merged**